helmfile/values/fluentd/aggregator-workload-cluster.yaml.gotmpl

```diff
@@ -0,0 +1,23 @@
+aggregator:
+  configMapFiles:
+    fluentd-labels.conf: |
+      # Relabel authlog
+      <match authlog.**>
+        @type relabel
+        @label @AUDIT
+      </match>
+
+      # Split kubeaudit
+      <match kubeaudit.**>
+        @type relabel
+        @label @AUDIT
+      </match>
+
+      # Relabel kubernetes
+      <match kubernetes.**>
+        @type relabel
+        @label @AUDIT
+      </match>
+
+    fluentd-filters.conf: |
```

Comment:
Should this be empty?

Reply:
No I will add so we can add filter through the config to capture AMS audit logs. I'm in the process of refactoring.